src/cntk-train/src/main/scala/CommandBuilders.scala

```diff
@@ -114,10 +120,8 @@ class MPICommandBuilder(log: Logger,
     "Identity file not found: "
 
   def runCommand(): Unit = {
-    val pathEnvVar = "PATH=/usr/bin/cntk/cntk/bin:$PATH"
-    val ldLibraryPathEnvVar =
-      "LD_LIBRARY_PATH=/usr/bin/cntk/cntk/lib:/usr/bin/cntk/cntk/dependencies/lib:$LD_LIBRARY_PATH"
-    val mpiArgs = s" -x $pathEnvVar -x $ldLibraryPathEnvVar --npernode ${nodeConfig.head._2} "
+    val exportClasspath = "export CLASSPATH=$($HADOOP_HOME/bin/hadoop classpath --glob); "
+    val mpiArgs = s" --npernode ${nodeConfig.head._2} "
     val cntkArgs = "cntk " + configs
       .map(c => if (fileBased) s"configFile=${configToFile(c)} " else c.text.mkString(" "))
       .mkString(" ")
```

Comment:
Please hoist the location of the SSH key pair into a constant somewhere. We might want to make it configurable.

Reply:
done